docs/third-party-developers/extensibility/data-store/checkout.md

```diff
@@ -0,0 +1,108 @@
+# wc/store/checkout
+
+## Table of Contents
+
+-   [Selectors](#selectors)
+    -   [getCustomerId](#getcustomerid)
+    -   [getCheckoutState](#getcheckoutstate)
+    -   [getOrderNotes](#getordernotes)
+    -   [hasError](#haserror)
+    -   [hasOrder](#hasorder)
+    -   [isAfterProcessing](#isafterprocessing)
+    -   [isBeforeProcessing](#isbeforeprocessing)
+    -   [isCalculating](#iscalculating)
+    -   [isComplete](#iscomplete)
+    -   [isIdle](#isidle)
+    -   [isProcessing](#isprocessing)
+
+## Selectors
+
+### getCustomerId
+
+Returns the ID of the customer.
```

Comment:
Could we specify that this is the WordPress user ID of the customer whose order the Checkout block is currently processing?

Reply:
Thanks for pointing this out, @opr. I just adjusted the wording according to your suggestion.